src/main/java/seedu/address/Main.java

```diff
@@ -25,17 +25,6 @@ public class Main {
     private static Logger logger = LogsCenter.getLogger(Main.class);
 
     public static void main(String[] args) {
-
-        // As per https://github.com/openjdk/jfx/blob/master/doc-files/release-notes-16.md
-        // JavaFX 16 (or later) runtime logs a warning at startup if JavaFX classes are loaded from
-        // the classpath instead of a module.
-        // Our application does not use Java modules yet. Even if it did, modules are ignored when
-        // packed into a FAT Jar file (as we do), which means this warning will persist even then.
-        // The warning however, can be safely ignored. Thus, the following log informs
-        // the user (if looking at the log output) that the said warning appearing in the log
-        // can be ignored.
-
-        logger.warning("The warning about Unsupported JavaFX configuration below can be ignored.");
```

Comment:
@itstrueitstrueitsrealitsreal So, this isn't applicable anymore? As per the comment, this is related to the JavaFX version rather than the Java version.

Reply:
@damithc Thanks for pointing them out, I mixed up the JavaFX version and the Java version. I'll revert this change.

Reply:
Interestingly, the JavaFX warning doesn't appear in Java 17. So, the comment could be wrong, and this warning should be removed. Worth investigating.

Reply:
For context, I am using `javafx.runtime.version: 22.0.1+101`. 

Looking at the JavaFX 16 release notes linked in the comment, it states that loading JavaFX classes from the classpath instead of modules will log a warning, which contradicts my testing.

This set of release notes also links to [this bug report](https://bugs.openjdk.java.net/browse/JDK-8256361) about the above mentioned warning, which aims to document this behavior. However, there are also some interesting comments from users on this issue.

One comment by [Gluon co-founder, Johan Vos](https://www.linkedin.com/in/johanvos/?originalSubdomain=be), summarizing this [email thread](https://mail.openjdk.org/pipermail/openjfx-dev/2020-April/025903.html) on the openjfx-dev mailing list stands out:

>The mail thread is very interesting. To give my summary of that thread:
>
>1. end users get errors (`no javafx.graphics module on modulepath`) that they don't understand and that seem pointless.
>
>2. there is however a very clear reason on why these warnings are there: JavaFX is designed to work as a set of modules (which of course does not mean that all JavaFX apps need to be modules too).
>
>3. At this moment, I am not aware of functionality that is broken when using the JavaFX JARs on the classpath, fuelling the assumption that the error is bogus.
>
>4. It is very likely though that some functionality is broken, or will be broken in the future (due to different access etc in a modular system), hence it is important we recommend users to use the JavaFX artifacts on the modulepath. It will be impossible to support the JavaFX platform jars running on the classpath, due to differences between packages/modules etc (which currently can mainly be overcome with e.g. add-opens etc).
>
>5. The javafx maven and gradle plugin already do this: if you use those, the javafx artifacts are on the modulepath.
>
>6. If you still use other tools/IDE's, or manually start JavaFX on the classpath, (a) that should be allowed, but (b) a clear warning is needed that, which explains that you might run into issues (now or later).
>
>This issue will deal with the last point (6b).

In particular, points 4 and 5 stand out, as they highlight that despite the errors seeming unintuitive and unfriendly to beginners, future changes might break the functionalities of JavaFX if users continue using JavaFX JARs on the classpath.

Point 5 is important because it points out that the JavaFX artifacts are already on the modulepath if the JavaFX Maven and Gradle plugins are already used, which is the case for AB3.

The actual email also outlines a workaround for the `no javafx.graphics module on modulepath` error, which is similar to what is currently being done in AB3 in `Main.java`.

Taking all this into account, we can either opt to:

### Keep the Warning

**Keeps the code future-proof:**
- Helps protect against future JavaFX updates that might reintroduce issues if classes are not loaded as modules.
- Aligns with community guidance and official documentation, preventing potential future problems.

**Maintenance:**
- Educates users on the recommended configuration for JavaFX and the significance of module usage.
- Provides clarity in logs, helping users quickly identify and understand the source of the warning.
- Helps to document past design decisions, helping others maintain the codebase in the future.

### Remove the Warning

**Not useful for current users:**
- The current setup with JavaFX 22 does not produce the warning, indicating the issue may be resolved or the configuration is correct.
- The JavaFX Maven and Gradle plugins already ensure compliance with best practices, making the warning redundant.

**YAGNI:**
- Removing outdated comments and warnings reduces code clutter and potential confusion for new developers.
- Simplifies logs, making it easier to spot actual issues that need attention.

**Minimizes verbosity:**
- Reduces the number of warnings which may lead to confusion even though they may not hinder functionality.

---

I believe that the best course of action is to keep the warning, given that it is in line with the official documentation of the maintainers of JavaFX, and the removal of the warning may lead to confusion down the line as future changes in the plugins bundled with JavaFX, such as Maven and Gradle, may lead to incompatibilities in the future.

Reply:
> I believe that the best course of action is to keep the warning, given that it is in line with the official documentation of the maintainers of JavaFX, and the removal of the warning may lead to confusion down the line as future changes in the plugins bundled with JavaFX, such as Maven and Gradle, may lead to incompatibilities in the future.

Thanks for the detailed investigation @itstrueitstrueitsrealitsreal 
Yes, we can keep it. In that case we should probably revise the text as `... below (if any) can be ignored.` (i.e., add `(if any)`) because the JavaFX warning might not appear in most cases.